variables.tf

```diff
@@ -301,7 +309,7 @@ variable "create_ecs_sg" {
   description = "Whether to create default security group for ECS or not."
 }
 
-variable "load_balancer_sg_ids" {
+variable "allowed_load_balancer_sg_ids" {
```

Comment:
might wanna update description to the effect that these are additional on top of any created SGs